overlord/snapstate/snapstate.go

```diff
@@ -349,14 +349,14 @@ func Info(s *state.State, name string, revision snap.Revision) (*snap.Info, erro
 	return nil, fmt.Errorf("cannot find snap %q at revision %s", name, revision.String())
 }
 
-// Current returns the information about the current revision of a snap with the given name.
-func Current(s *state.State, name string) (*snap.Info, error) {
+// CurrentSideInfo returns the information about the current revision of a snap with the given name.
+func CurrentSideInfo(s *state.State, name string) (*snap.Info, error) {
```

Comment:
this returns a snap.Info so ti cannot be called that,  there are arguments to call it CurrentInfo maybe